src/integration-test/java/org/mskcc/kickoff/fast/endtoend/MappingFilePrinterTest.java

```diff
@@ -0,0 +1,195 @@
+import org.mskcc.kickoff.printer.observer.ObserverManager;
+import org.mskcc.kickoff.resolver.PairednessResolver;
+import org.mskcc.kickoff.retriever.ReadOnlyExternalSamplesRepository;
+import org.mskcc.kickoff.retriever.RequestDataPropagator;
+import org.mskcc.kickoff.sampleset.SampleSetProjectInfoConverter;
+import org.mskcc.kickoff.sampleset.SampleSetToRequestConverter;
+import org.mskcc.kickoff.util.Constants;
+import org.mskcc.kickoff.validator.ErrorRepository;
+import org.mskcc.kickoff.validator.PairednessValidPredicate;
+import org.mskcc.kickoff.velox.RequestsRetrieverFactory;
+import org.mskcc.kickoff.velox.VeloxConnectionData;
+import org.mskcc.kickoff.velox.VeloxProjectProxy;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.*;
+import java.util.function.BiPredicate;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mskcc.kickoff.util.Utils.sampleNormalization;
+
+@RunWith(Parameterized.class)
+public class MappingFilePrinterTest {
+
+    private static final Logger DEV_LOGGER = Logger.getLogger(Constants.DEV_LOGGER);
+    private static final String connectionFile = "/lims-tango-dev.properties";
+    private static final String connectionFileTest = "/lims-tango-test.properties";
+    private static final String fastq_path="/ifs/archive/GCL";
+    private static final String designFilePath = "/ifs/projects/CMO/targets/designs";
+    private static final String resultsPathPrefix = "/ifs/solres/seq";
+
+    private final String projectId;
+    private KickoffRequest request;
+
+
+    private ProjectFilesArchiver archiverMock = mock(ProjectFilesArchiver.class);
+    private ProjectInfoRetriever projInfoRetriever = new ProjectInfoRetriever();
+    private RequestDataPropagator reqDataPropagator = new RequestDataPropagator(designFilePath, resultsPathPrefix,
+            mock(ErrorRepository.class), (bs1, bs2) -> true);
+    private SampleSetProjectInfoConverter projInfoConv = new SampleSetProjectInfoConverter();
+    private SampleSetToRequestConverter sampleSetToReqConv = new SampleSetToRequestConverter(
+            projInfoConv,
+            (s1, s2) -> true,
+            mock(ErrorRepository.class));
+    private RequestsRetrieverFactory requestsRetrieverFactory = new RequestsRetrieverFactory(projInfoRetriever,
+            reqDataPropagator, reqDataPropagator, sampleSetToReqConv, mock(ReadOnlyExternalSamplesRepository.class),
+            mock(BiPredicate.class), mock(BiPredicate.class), mock(ErrorRepository.class));
+    private VeloxProjectProxy veloxProjectProxy;
+    private MappingFilePrinter mappingFilePrinter;
+    private VeloxConnection connection;
+
+    public
+    MappingFilePrinterTest(String projectId) {
+        this.projectId = projectId;
+    }
+
+    @Before
+    public void
```

Comment:
I see new lines in few places, is it somehting with formatting?

Reply:
it's just some autoformating that put method name in separate line from labels and access modifiers. i am reformating it.